error-handler.js

```diff
@@ -269,6 +277,9 @@ createHandler = function createHandler(options) {
       return resumeOrClose(status);
     }
 
+    // Always set a response status.
```

Comment:
NO.

This will throw an error and crash the server if the response has already been sent.